Logpipelines ext (#1430)
kyma-project/busola: docs/extensibility/display-widgets.md

### CodeViewer

> CodeViewer widgets display values using a read-only code editor. The editor will autodetect the language.

Reviewer: ```suggestion
CodeViewer widgets display values using a read-only code editor. The editor autodetects the language.
```

  "children": [{ "path": "name" }, { "path": "status" }]
}
```

> ### ResourceRef

Reviewer: ```suggestion
### ResourceRefs
```

ResourceRefs widgets render the lists of links to the associated resources. The corresponding specification object must be an array of objects `{name: 'foo', namespace: 'bar'}`.

> Additionally define the kind of the linked resources by passing resource Kubernetes `kind`.

Reviewer: ```suggestion
Additionally, you must define the kind of the linked resources by passing the Kubernetes resource `kind` (for example, `Secret`, `ConfigMap`).
```
